NoteFind/View Controllers/NewNoteViewController.m

```diff
@@ -47,9 +43,12 @@ - (void)setImportMenu:(UIMenu *)importMenu {
     
     [options addObject:[UIAction actionWithTitle:@"take picture w/ camera" image:nil identifier:nil handler:^(__kindof UIAction * _Nonnull action) {
         NSLog(@"button was tapped successfully!");
+        CameraImport *cameraImport = [[CameraImport alloc]init];
```

Comment:
The camera import object instance will fall out of memory if not retained here and likely will lead to issues in this flow. You probably want to create and retain an instance of `CameraImport` as a `@property` early in this view controller (like `viewDidLoad:`)